packages/storybook-ui/src/modules/ui/components/down_panel/style.js

```diff
@@ -26,6 +26,11 @@ export default {
   tabbar: {
     display: 'flex',
     flexWrap: 'wrap',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    maxHeight: 60,
```

Comment:
When this at the right side and there are a lot of tabs - they go in several rows. So maxHeight will break it. Better to put it to the tab style individually.